bst.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var recursion = function(root, min, max) {
+    if (!root) return true;
+    return root.val < max && root.val > min
```

Comment:
A ternary operator can be replaced with `&&` here

Reply:
I meant something different ;)
You can write this without ternary operators, using just `&&` operator.